src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,51 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  // static alive = [];
+
+  constructor(health = 100, name) {
```

Comment:
```suggestion
  constructor(name, health = 100) {
```
let's put obligate property in the first place